docs/getting-started/create-configuration.md

```diff
@@ -657,6 +666,12 @@ metadata:
   annotations:
     guide: quickstart
     provider: azure
+spec:
+  crossplane:
+    version: ">=v1.0.0-0"
+  dependsOn:
+    - provider: crossplane/provider-azure
+      version: ">=v0.13.0"
 ```
 
 ```console
```

Comment:
I think we can separate `push` command from this snippet to have its own because pushing requires credentials to be set. Also, `build` doesn't let you specify a tag like `docker build`, hence only way to push the configuration is to be in that folder. It might be worth mentioning that you tag the configuration when you push it, not during the build. We might also mention it never ends up in your local docker registry, but that may be too much information, though helpful to know if you use local cluster like `kind` to test the package.

Reply:
These are excellent points, I think they probably necessitate a separate document or additions to the package doc in `Concepts`. Would be happy to follow up with that 👍